opmd_viewer/openpmd_timeseries/main.py

```diff
@@ -347,7 +347,7 @@ def get_particle(self, var_list=None, species=None, t=None, iteration=None,
             return(data_list)
 
     def get_field(self, field=None, coord=None, t=None, iteration=None,
-                  m='all', theta=0., slicing=0., slicing_dir='y',
+                  m='all', theta=0., slicing=None, slicing_dir=None,
```

Comment:
Having `y` as the default slicing direction was kind of arbitrary. Now that slicing is also allowed in 2d, it can lead to an issue.
Depending on the code, 2d simulations can rely on any set of 2 coordinates in `x,y,z`. If the code uses `x-z` as coordinates for 2d simulations, `get_field` returns a 2d array by default. If the code uses `x-y` instead, `get_field` returns a 1d array by default.
To avoid this issue, @RemiLehe and I agreed to change the behavior of `get_field` so that no slicing is done by default. However, it breaks backward compatibility, so it may impact some users.

@soerenjalas @hightower8083 @ax3l is this fine for you?

Reply:
Just to clarify @MaxThevenet's comment: the backward-compatibility breaks exclusively in the case where the user calls `get_field` **without explicitly setting the `slicing_dir` argument** and **for a 3D dataset**. 

For instance
```
E, info = ts.get_field('E', 'x')
```
used to return `E` as a 2d array (arbitrarily sliced along 'y'), but with this PR it returns a 3d array (and the user needs to explicitly set `slicing_dir` in order to get a 2d slice).

For 2d and circ dataset, the backward-compatibility is preserved.

Reply:
No problem with me -- the proposed behaviour is intuitively correct. 
By the way, for my curiosity, are there plans for the z-silcing in the multi-mode circ? 
Its probably kinda a rarely needed thing (guess can be done in 3d via Visit), but nevertheless at some point this could be nice to have, in particular to see propagation of laser with a realistic profile.

Reply:
That's fine, just make sure to release it into a `0.9` or `1.0` version so people see the possible break.

@MaxThevenet you can also already add this breaking behavior to the `CHANGELOG.md` file.